src/NexusMods.App.UI/Controls/Navigation/NavigationMenuItem.cs

```diff
@@ -41,8 +40,8 @@ public NavigationMenuItem()
 
         _contextMenuCommand = ReactiveCommand.Create<OpenPageBehaviorType>(openPageBehaviorType =>
         {
-            NavigationInformation = NavigationInformation.From(openPageBehaviorType);
-            OnClick(new RoutedEventArgs());
+            var navigationInformation = NavigationInformation.From(openPageBehaviorType);
+            NavigationCommand?.Execute(navigationInformation).Subscribe();
```

Comment:
The proper solution would be to do what the `NavigationControl` does and hook into the pointer pressed/released events, but this is fine for now.

Reply:
Btw, should I be disposing that subscription?